src/main/java/core/camera/CameraManager.java

```diff
@@ -39,7 +40,8 @@ public void step(float dt) {
         }
 
         Vector3 dp = new Vector3(subjectPosition.x, subjectPosition.y, 0).sub(camera.position);
-        Function<Float, Float> f = x -> (float) Math.round((x - Math.tanh(x) + 0.1 * Math.tanh(10 * x)) / unitScale * 2) * unitScale / 2;
+        Function<Float, Float> f = x ->
```

Comment:
Can you comment this more to explain exactly what's going on for future reference

Reply:
done in f9d0aa5